constants.json

```diff
@@ -1,7 +1,8 @@
 {
-    "baseURL": "/management/",
+    "baseURL": "https://nightly.gravitee.io/management/",
```

Comment:
Be careful, this must not be part of the bundle.

FYI, you can now execute the UI pointing on nightly rest-api by executing : `npm run serve:nightly`